master/buildbot/www/hooks/github.py

```diff
@@ -170,6 +170,41 @@ def handle_pull_request(self, payload, event):
             len(changes), number))
         return changes, 'git'
 
+    def get_changes(self, category, payload):
```

Comment:
create event need another set of json , so kept it in separate functions which wont disturb other events.